2D3D_UnityProject/Assets/Scripts/Player/Perspective/PerspectiveController.cs

```diff
@@ -101,6 +101,14 @@ public void UpdatePerspective(Actor player)
             // If button is being pressed
             if (Input.GetKeyDown(key)) 
             {
+                if(key == KeyCode.Alpha2)

```

Comment:
Can you change this to check for the top-down perspective instead of just keypress (since that could be 

On line 113 you're getting the CameraView enum that tells you this, so just check it's the top-down camera view